group_project/public/js/group_project.js

```diff
@@ -52,8 +52,11 @@ function GroupProjectBlock(runtime, element) {
         fill_field = $('#assess_' + data_item, section_node);
       }
       var data_class = fill_field.data('class');
+      // Remove old feedback data
+      fill_field.find('.feedback-data').remove();
```

Comment:
It looks like line 46 above should be doing the clearing of all of these fields. Seems like the author forgot to include ```class="answer-value"``` in these nodes, otherwise this change would be unnecessary. 

Having said that, this change seems like a cleaner way to remove the existing answers so we should remove the requirement that they attach the ```class="answer-value"``` by removing it from the default document and the script up above.

However, I'm a little worried that there might be cases where an item is removed and the prior data remains because we are cleaning one-by-one based upon the data present - if there is an item that was there and is no more, then the UI will leave it as the value previously given (I doubt that this situation will ever happen, but as the code changes people may not know about this). I think it is safer to remove all data in the given section, because we are reloading all data for that section.

So, I think that instead of removing each as data comes in I think it would be better to 
# replace line 46 with ```$('. feedback-data', section_node).remove();``` to clear all items in the section
# remove line 55 & 56
# remove ```class="answer-value"``` from the default document
# tell the client to remove the document requirement for ```class="answer-value"```